_aggregations/metric/cardinality.md

```diff
@@ -28,22 +44,53 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
 ```
 {% include copy-curl.html %}
 
-#### Example response
+## Example response
+
+The aggregation returns the following:
 
 ```json
-...
-  "aggregations" : {
-    "unique_products" : {
-      "value" : 7033
+{
+  "took": 176,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 4675,
+      "relation": "eq"
+    },
+    "max_score": null,
+    "hits": []
+  },
+  "aggregations": {
+    "unique_products": {
+      "value": 7033
     }
   }
 }
 ```
 
-Cardinality count is approximate.
-If you have tens of thousands of products in your hypothetical store, an accurate cardinality calculation requires loading all the values into a hash set and returning its size. This approach doesn't scale well; it requires huge amounts of memory and can cause high latencies.
+## Controlling precision
+
+An accurate cardinality calculation requires loading all the values into a hash set and returning its size. This approach doesn't scale well; it can require huge amounts of memory and cause high latencies.
+
+You can control the trade-off between memory and accuracy with the `precision_threshold` setting. This parameter sets the threshold below which counts are expected to be close to accurate. Counts higher than this value might be less accurate.
 
-You can control the trade-off between memory and accuracy with the `precision_threshold` setting. This setting defines the threshold below which counts are expected to be close to accurate. Above this value, counts might become a bit less accurate. The default value of `precision_threshold` is 3,000. The maximum supported value is 40,000.
+The default value of `precision_threshold` is 3,000. The maximum supported value is 40,000.
+
+The cardinality aggregation uses the [HyperLogLog++ algorithm](https://static.googleusercontent.com/media/research.google.com/fr//pubs/archive/40671.pdf). Cardinality counts are typically very accurate up to the precision threshold, and are within 6% of the true count in most other cases, even with a threshold as low as 100.
+
+### Pre-computing hashes
+
+For high-cardinality string fields, storing hash values for the field in the index and computing the cardinality of the hash can save compute and memory resources. Use this approach with caution; it is more efficient only for sets with long strings and/or high cardinality. Numeric fields and less taxing string sets are better processed directly.
```

Comment:
"for the field in the index" => "for the index field"? Please change "less taxing" to something like "less complex".

Reply:
```suggestion
For high-cardinality string fields, storing hash values for the index field and computing the cardinality of the hash can save compute and memory resources. Use this approach with caution; it is more efficient only for sets with long strings and/or high cardinality. Numeric fields and less memory-consuming string sets are better processed directly.
```